tests/tracking/test_model_registry.py

```diff
@@ -325,6 +325,86 @@ def test_set_registered_model_tag_with_empty_string_as_value(mlflow_client):
     assert {"tag_key": ""}.items() <= mlflow_client.get_registered_model(name).tags.items()
 
 
+@pytest.mark.parametrize("max_results", [1, 8, 100])
+@pytest.mark.parametrize(
+    ("filter_string", "filter_func", "order_by", "order_by_key", "order_by_desc"),
+    [
+        (None, lambda mv: True, None, None, False),
+        ("", lambda mv: True, ["name DESC"], lambda mv: mv.name, True),
+        (
+            "name LIKE '%2'",
+            lambda mv: mv.name.endswith("2"),
+            ["version_number DESC"],
+            lambda mv: mv.version,
+            True,
+        ),
+        ("name ILIKE '%rm%00%'", lambda mv: "00" in mv.name, None, None, False),
+        ("name LIKE '%rm%00%'", lambda mv: False, None, None, False),
+        ("name = 'badname'", lambda mv: False, None, None, False),
+        (
+            "name = 'CreateRMsearchForMV03'",
+            lambda mv: mv.name == "CreateRMsearchForMV03",
+            None,
+            None,
+            False,
+        ),
+    ],
+)
+def test_search_model_versions_flow_paginated(
```

Comment:
Separating test cases like below might be easier to understand:

```python
def test_search_model_versions_filter_string():
    ...

def test_search_model_versions_order_by():
    ...

def test_search_model_versions_order_max_results():
    ...

```

Reply:
+1 that test parametrize is pretty dense.

Reply:
Done.